packages/orbit-components/src/Box/index.js.flow

```diff
@@ -0,0 +1,147 @@
+
+export type ColorTokens =
+  | "productLight"
+  | "productLightHover"
+  | "productLightActive"
+  | "productNormal"
+  | "productNormalHover"
+  | "productNormalActive"
+  | "productDark"
+  | "productDarkHover"
+  | "productDarkActive"
+  | "productDarker"
+  | "white"
+  | "whiteHover"
+  | "whiteActive"
+  | "cloudLight"
+  | "cloudLightHover"
+  | "cloudLightActive"
+  | "cloudNormal"
+  | "cloudNormalHover"
+  | "cloudNormalActive"
+  | "cloudDark"
+  | "inkLighter"
+  | "inkLighterHover"
+  | "inkLighterActive"
+  | "inkLight"
+  | "inkLightHover"
+  | "inkLightActive"
+  | "inkNormal"
+  | "inkNormalHover"
+  | "inkNormalActive"
+  | "orangeLight"
+  | "orangeLightHover"
+  | "orangeLightActive"
+  | "orangeNormal"
+  | "orangeNormalHover"
+  | "orangeNormalActive"
+  | "orangeDark"
+  | "orangeDarkHover"
+  | "orangeDarkActive"
+  | "orangeDarker"
+  | "redLight"
+  | "redLightHover"
+  | "redLightActive"
+  | "redNormal"
+  | "redNormalHover"
+  | "redNormalActive"
+  | "redDark"
+  | "redDarkHover"
+  | "redDarkActive"
+  | "redDarker"
+  | "greenLight"
+  | "greenLightHover"
+  | "greenLightActive"
+  | "greenNormal"
+  | "greenNormalHover"
+  | "greenNormalActive"
+  | "greenDark"
+  | "greenDarkHover"
+  | "greenDarkActive"
+  | "greenDarker"
+  | "blueLight"
+  | "blueLightHover"
+  | "blueLightActive"
+  | "blueNormal"
+  | "blueNormalHover"
+  | "blueNormalActive"
+  | "blueDark"
+  | "blueDarkHover"
+  | "blueDarkActive"
+  | "blueDarker"
+  | "socialFacebook"
+  | "socialFacebookHover"
+  | "socialFacebookActive";
+
+export type SpacingToken =
+  | "none"
+  | "XXXsmall"
+  | "XXsmall"
+  | "XSmall"
+  | "small"
+  | "medium"
+  | "large"
+  | "Xlarge"
+  | "XXlarge"
+  | "XXXlarge";
+
+export type SpacingObject = {|
+  +left?: SpacingToken,
+  +top?: SpacingToken,
+  +right?: SpacingToken,
+  +bottom?: SpacingToken,
+|};
+
+export type MediaQueryObject = $Diff<
+  // eslint-disable-next-line no-use-before-define
+  Props,
+  {| as: *, children: *, mediumMobile: *, largeMobile: *, tablet: *, desktop: *, largeDesktop: * |},
+>;
+
+export type Props = {|
+  +as?: string,
+  +padding?: SpacingObject | SpacingToken,
+  +margin?: SpacingObject | SpacingToken,
+  +display?: "none" | "flex" | "inline-flex" | "block" | "inline" | "inline-block",
+  +wrap?: "wrap" | "nowrap",
+  +shrink?: 0 | 1,
+  +grow?: 0 | 1,
+  +direction?: "row" | "column" | "row-reverse" | "column-reverse",
+  +width?: "full" | "auto",
+  +maxWidth?: string,
+  +height?: "full" | "auto",
+  +maxHeight?: string,
+  +position?: "absolute" | "relative" | "fixed",
+  +left?: string,
+  +top?: string,
+  +right?: string,
+  +bottom?: string,
+  +align?: "start" | "end" | "center" | "stretch",
+  +justify?: "center" | "start" | "end" | "between" | "around",
+  +textAlign?: "left" | "right" | "center",
+  +elevation?: Elevation,
+  +color?: ColorTokens,
+  +background?: ColorTokens,
+  +borderRadius?: "small" | "normal" | "large" | "radiusCircle",
```

Comment:
The use of `radiusCircle` won't work, because the final normalized token name will be `borderRadiusRadiusCircle`.

It should use the `borderRadiusCircle` token, hence the short name should be `circle`.

Don't forget to update it in all places.